### PR TITLE
chore: Update README and camelCase all variables

### LINKED ISSUE #6: Update README About Security Review
The top `README.md` should have a clearer note about how this library has not had a security review and currently only be used for testing pruposes.

This does exist in the notes but it should be the first line, be bold text and have some :warning: emojis

### LINKED ISSUE #8: Solidity Variables Should All Be camelCase
There's a number of local variabels and function parameters which use `snake_case` we should follow the Solidity style guide and use `camelCase`

### PR DESCRIPTION
**Pull request by kamuikatsurgi**
Closes #6 
Closes #8

### CODE REVIEW

**Comment by kamuikatsurgi** [comment]
image

`> [!WARNING]`  Will look like this.

File: README.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
-# Merkle Multiproof Inputs Generation For Forge
+> [!WARNING]
+> Note that this library has not had a security review yet, is not gas-efficient, and should only be used for testing.
 
-This is a simple library to generate Merkle Multiproof inputs for OpenZeppelin's `MerkleProof` library. It is written in Solidity and can be used in Forge framework. The library has been tested to work with at most 10k of arbitrary leaves and arbitrary indices of any size.
+# Merkle MultiProof and SingleProof Inputs Generation For Forge
 
-Note that this library is not gas-efficient and should be used for testing purposes only.
+This simple library generates Merkle MultiProof and SingleProof inputs for OpenZeppelin's `MerkleProof` library. It is written in Solidity and can be used in the Forge framework. The library has been tested to work with at most 10k arbitrary leaves and arbitrary indices of any size.
```
Comment (kirk-baird) [inline README.md]: I think it'd be a good idea to link the OZ code and also note the version of OZ we're using. Some of the older version of OZ merkle proof wouldn't be compatible.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,27 @@
...
 
 ## Usage
 
-See `MerkleGen.t.sol` for a sample. First, deploy `MerkleGen` contract. Then, prepare the leaves and indices. Finally, call `gen()` to generate the necessary inputs.
+See `MerkleGen.t.sol` for example:
```
Comment (kirk-baird) [inline README.md]: Let's make this a link to the test file for ease. I believe the syntax is something like
```
[MerkleGen.t.sol](./test/MerkleGen.t.sol)
```

**Review by kirk-baird — APPROVED** [review]
I've left two minor comments on the `README.md` that should be quick to fix. After fixing those I'm happy for you to merge :)

Thank you again for all the work on this!

**Pull request merged**